src/Parallel-Solvers/Simulation_Parameters/Geometry.h

```diff
@@ -19,35 +31,137 @@ struct Volume : Yaml::ValidatedYaml {
     double radius2 = 0;
 
     double x1, x2, y1, y2, z1, z2;
-
+    
+    std::string stl_file_path;
+    size_t num_voxel_x;
+    size_t num_voxel_y;
+    size_t num_voxel_z;
+    mtr::CArray<bool> voxel_elem_values;
+    std::string vtk_file_path;
+    double orig_x = 0;
+    double orig_y = 0;
+    double orig_z = 0;
+    double voxel_dx;
+    double voxel_dy;
+    double voxel_dz;
+    double length_x = 0;
+    double length_y = 0;
+    double length_z = 0;
+    double i0_real;
+    double j0_real;
+    double k0_real;
+    int i0;
+    int j0;
+    int k0;
+    int elem_id0;
+    bool fill_this;
+    
+    KOKKOS_FUNCTION
+    // Run voxelization scheme on stl file
```

Comment:
@nathanielmorgan Do you know if having a comment after "KOKKOS_FUNCTION" breaks anything?  My guess is no, but it feels off.

Reply:
Moved the comment so that it is before "KOKKOS_FUNCTION."

Reply:
Thanks.  Sorry for the nit picking here.  I was mostly just curious.  It seems it does work. :)